app/test/integration_test.go

```diff
@@ -206,78 +201,6 @@ func (s *IntegrationTestSuite) TestMaxBlockSize() {
 	}
 }
 
-func (s *IntegrationTestSuite) TestSubmitPayForBlob() {
```

Comment:
[question] was this test moved or just deleted? It doesn't look like the original PR deleted them: https://github.com/celestiaorg/celestia-app/pull/2455/files#diff-d66f948da147c1b292dcf8131bd525c02c6e23daa6c64712a85e53476271cc65L60

Reply:
it was deleted like in this pr https://github.com/celestiaorg/celestia-app/pull/2834/files#diff-d66f948da147c1b292dcf8131bd525c02c6e23daa6c64712a85e53476271cc65

Reply:
Thanks. Resolving